aws/logs_monitoring/template.yaml

```diff
@@ -65,7 +65,7 @@ Parameters:
     AllowedValues:
       - true
       - false
-    Description: By default, the forwarder sends logs using HTTPS through the port 443. To send logs over a SSL encrypted TCP, set this parameter to true.
+    Description: By default, the forwarder sends logs using HTTPS through the port 443. To send logs over an SSL encrypted TCP, set this parameter to true.
```

Comment:
@tianchu I updated some of the verbiage in the CloudFormation template to match @nhinsch's suggestions. Does this merit a new release, or should we wait and bundle these changes in with the next release?